### PR TITLE
Add views and routes for Organization model

### PR DESCRIPTION
**Pull request by AI-Mozi**
#78 #79

### CODE REVIEW

**Comment by AI-Mozi** [comment]
@postmodern Could you pleas check if `show` view works correctly for you? (I had some problems locally).
I'm also not sure if I've added all necessary fields.

**Comment by postmodern** [comment]
@AI-Mozi fixed an issue in `ronin-db-activerecord` and had to recreate my database. Probably should change `./scripts/server` to use a local database in `db/` to make development easier?

Here is what a new `Organization` with just a name looks like:
localhost_1337_db_organizations_1

File: views/db/organizations/show.erb [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    <tr>
+      <td><strong>Type:</strong></td>
+      <td><%=h @organization.type %></td>
+    </tr>
+
+    <tr>
+      <td><strong>Departments:</strong></td>
+      <td>
+        <% @organization.departments.each do |department| %>
+          <p><a href="/db/departments/<%=h department.id %>"><%=h department %></a></p>
```
Comment (postmodern) [inline views/db/organizations/show.erb:24]: Should be named `/db/organization_departments/:id` (if we want to link directly to departments, without knowing their parent org ID) or `/db/organizations/:id/departments/:dept_id`.

File: views/db/organizations/show.erb [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+          <p><a href="/db/phone_numbers/<%=h phone_number.id %>"><%=h phone_number %></a></p>
+        <% end %>
+      </td>
+    </tr>
+
+    <tr>
+      <td><strong>Members:</strong></td>
+      <td>
+        <% @organization.members.each do |member| %>
+          <p><a href="/db/members/<%=h member.id %>"><%=h member %></a></p>
```
Comment (postmodern) [inline views/db/organizations/show.erb:60]: Same here, should be either `/db/organization_members/:id` or `/db/organizations/:id/members/:member_id`.

File: views/db/organizations/show.erb [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        <% end %>
+      </td>
+    </tr>
+
+    <tr>
+      <td><strong>Customers:</strong></td>
+      <td>
+        <% @organization.customers.each do |customer| %>
+          <p><a href="/db/customers/<%=h customer.id %>"><%=h customer %></a></p>
```
Comment (postmodern) [inline views/db/organizations/show.erb:69]: `Organization#customers` returns `Array<Person>`, so we could link directly to `/db/people/:id`.

File: views/db/organizations/show.erb [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+        <% end %>
+      </td>
+    </tr>
+
+    <tr>
+      <td><strong>Vendors:</strong></td>
+      <td>
+        <% @organization.vendors.each do |vendor| %>
+          <p><a href="/db/vendors/<%=h vendor.id %>"><%=h vendor %></a></p>
```
Comment (postmodern) [inline views/db/organizations/show.erb:78]: `Organization#vendors` returns `Array<Organization>`, so we could link directly to `/db/organizations/:id`.

File: views/db/organizations/show.erb [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+    <tr>
+      <td><strong>Created:</strong></td>
+      <td><%=h @organization.created_at %></td>
+    </tr>
+  </tbody>
+</table>
+
+<%= partial(:notes, notes: @organization.notes) %>
+
```
Comment (postmodern) [inline views/db/organizations/show.erb:109]: Extra newline.

**Review by postmodern — CHANGES_REQUESTED** [review]
Some suggestions on the links/routes to other models associated with `Organization`.

**Review by postmodern — APPROVED** [review]

**Pull request merged**